examples/popup-directive/index.ts

```diff
@@ -0,0 +1,106 @@
+// Shims
+import 'reflect-metadata';
+import 'zone.js';
+
+import { YagaModule } from '../../lib/index'; // @yaga/leflet-ng2
+
+import { Component, PlatformRef } from '@angular/core';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { Point, LatLng } from 'leaflet';
+
+import { ExampleAppComponentBlueprint, IExampleProperties } from '../app-component-blueprint';
+import { ExamplePropertiesModule, PROPERTIES_WRAPPER } from '../property.component';
+
+const platform: PlatformRef = platformBrowserDynamic();
+
+/* tslint:disable:max-line-length */
+const template: string = `
+<example-header [title]="'Popup-Directive'"></example-header>
+<div class="container">
+  <div class="map">
+    <yaga-map>
+      <yaga-marker>
```

Comment:
@atd-schubert Do we have a default marker position?

Reply:
Yes it is [0, 0], but we should provide it to make not a bad example...